### PR TITLE
Release/4.1.2

### PR DESCRIPTION
**Pull request by mcfadden8**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,8 +40,6 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 - Added more compiler configurations to GitLab CI.
```
Comment (davidbeckingsale) [inline CHANGELOG.md:41]: This is in the wrong place.

**Review by davidbeckingsale — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mcfadden8) [reply CHANGELOG.md:41]: Yeah.  It looks like it has been in the wrong place even since 4.1.1.  Strange

Comment (davidbeckingsale) [reply CHANGELOG.md:41]: It wasn't in 4.1.1

**Comment by mcfadden8** [comment]
LGTM

**Review by davidbeckingsale — APPROVED** [review]

**Comment by davidbeckingsale** [comment]
LGTM

**Pull request merged**